consul/map.jinja

```diff
@@ -13,10 +15,9 @@
         }
   }
   ,grain="cpuarch"
-  ,merge=salt['pillar.get']('consul'))
+  ,merge=consul)
 %}
-{% do defaults.consul.update(cpu_arch_map) %}
 
-{% set consul = salt['pillar.get']('consul', default=defaults.consul, merge=True) %}
+{% do consul.update(cpu_arch_map) %}
```

Comment:
I guess that since `cpu_arch_map` is a result of merging architectures, Pillar values and defaults using `grains.filter_by` function, it already contains a full data set for the formula, right? So no need to call `update()` method, just to assign it back to `consul` variable. Or even use `consul` as a result of the above statement and skip dealing with `cpu_arch_map` variable at all?

I mean this:
```jinja
{% set consul = salt['grains.filter_by']({
        'armv6l': {
            "arch": 'arm'
        },
        'armv7l': {
            "arch": 'arm'
        },
        'x86_64': {
            "arch": 'amd64'
        }
  }
  ,grain="cpuarch"
  ,merge=consul)
%}
```